server.js

```diff
@@ -2940,50 +2940,12 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
-// dotnet-status integration.
+// dotnet-status integration - deprecated as of April 2018.
```

Comment:
I know we're doing this with immediate effect so we don't need to rely on `isDeprecated()`, but for completeness could we put an entry in https://github.com/badges/shields/blob/master/lib/deprecated-services.js . That way the info that tells us when badge code can be completely removed is all in the same place.

Reply:
I had not actually noticed that file, done. ^^